packages/core/content-type-builder/admin/src/components/AttributeIcon/index.js

```diff
@@ -68,8 +78,13 @@ const AttributeIcon = ({ type, ...rest }) => {
   return <IconBox as={Compo} {...rest} />;
 };
 
+AttributeIcon.defaultProps = {
+  customField: null,
+};
+
 AttributeIcon.propTypes = {
   type: PropTypes.string.isRequired,
+  customField: PropTypes.string,
```

Comment:
What do you think about changing the name to `customFieldUid`, for me `customField` suggests the value not the key.

Reply:
I think we should keep it consistent if you change the name for the attributeIcon you'll need to do it for the listRow as well (where the property is spread)

Reply:
I understand the concern, but since "customField" is coming from the schema.json, I don't think it's worth trying to rename it everywhere on the CTB. I'd rather have data with a consistent shape like Soupette mentioned

Reply:
I completely agree!

Reply:
I hear what you are saying about consistency in the CTB, but I’m not sure why the naming should be based on the `schema.json` shape. In the core we call this a `uid` and the object registered a `customField` so either way I see a bit of an inconsistency. But this works for me it was really just a small suggestion for readability.